src/Tests/Microsoft.NET.Build.Tests/GivenThatWeWantToBuildALibrary.cs

```diff
@@ -786,6 +790,7 @@ public void It_passes_ridless_target_to_compiler()
                 .Pass();
         }
 
+        // TODO: need to set TargetPlatformSupported in MSBuild.SDK.Extras for this to pass
```

Comment:
So is this test failing now?  If so, it seems like we should have it marked with a Skip.  If not, does it really need o TODO?

Reply:
A commit from the original PR got lost, I've added it back now so tests should pass :)